<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>administracao_pedido</class>
 <widget class="QMainWindow" name="administracao_pedido">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de pedidos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Lista</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_pedidos">
      <property name="geometry">
       <rect>
        <x>-3</x>
        <y>48</y>
        <width>402</width>
        <height>182</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>7</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>-7</x>
        <y>10</y>
        <width>397</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_pesquisar">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="inputMask">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_pesquisar">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Pesquisar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_11">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>240</y>
        <width>381</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButton_novo">
         <property name="text">
          <string>Novo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_gravar">
         <property name="text">
          <string>Gravar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_voltar">
         <property name="text">
          <string>Voltar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Administração</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_id_pedido">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>47</width>
        <height>24</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>9999</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>272</x>
        <y>46</y>
        <width>120</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Fornecedor</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>46</y>
        <width>120</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Cliente</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_12">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>240</y>
        <width>381</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="pushButton_novo_2">
         <property name="text">
          <string>Novo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_gravar_2">
         <property name="text">
          <string>Gravar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_voltar_2">
         <property name="text">
          <string>Voltar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_cliente">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>70</y>
        <width>40</width>
        <height>24</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>999</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_cliente">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Vincular</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_fornecedor">
      <property name="geometry">
       <rect>
        <x>272</x>
        <y>70</y>
        <width>40</width>
        <height>24</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="inputMask">
       <string>999</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_fornecedor">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>70</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Vincular</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>166</y>
        <width>120</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Data do pedido</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_data">
         <property name="inputMask">
          <string>99/99/9999</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>106</y>
        <width>251</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Descrição</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit_descricao"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>142</x>
        <y>46</y>
        <width>120</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Funcionario</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_funcionario">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Vincular</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_funcionario">
      <property name="geometry">
       <rect>
        <x>142</x>
        <y>70</y>
        <width>40</width>
        <height>24</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="inputMask">
       <string>999</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>107</y>
        <width>120</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Valor do pedido</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_valor_pedido">
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>133</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="radioButton_venda">
         <property name="text">
          <string>Venda</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_compra">
         <property name="text">
          <string>Compra</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Produtos</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_produtos">
      <property name="geometry">
       <rect>
        <x>-3</x>
        <y>70</y>
        <width>402</width>
        <height>160</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>8</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QWidget" name="layoutWidget_10">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>240</y>
        <width>381</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_gravar_produto">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Adicionar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_novo_produto">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Limpar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_remover">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Remover</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>8</y>
        <width>158</width>
        <height>55</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Quantidade</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_estoque_quantidade">
         <property name="inputMask">
          <string>9999</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_produto">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>10</y>
        <width>40</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_selecionar_produto">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>40</y>
        <width>201</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Selecionar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>56</x>
        <y>12</y>
        <width>50</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Produto</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>165</x>
        <y>12</y>
        <width>50</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Estoque</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_estoque">
      <property name="geometry">
       <rect>
        <x>117</x>
        <y>10</y>
        <width>40</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEdit_pesquisar</tabstop>
  <tabstop>pushButton_pesquisar</tabstop>
  <tabstop>tableWidget_pedidos</tabstop>
  <tabstop>pushButton_novo</tabstop>
  <tabstop>pushButton_gravar</tabstop>
  <tabstop>pushButton_voltar</tabstop>
  <tabstop>lineEdit_id_pedido</tabstop>
  <tabstop>lineEdit_id_cliente</tabstop>
  <tabstop>pushButton_cliente</tabstop>
  <tabstop>lineEdit_id_fornecedor</tabstop>
  <tabstop>pushButton_fornecedor</tabstop>
  <tabstop>lineEdit_id_funcionario</tabstop>
  <tabstop>pushButton_funcionario</tabstop>
  <tabstop>plainTextEdit_descricao</tabstop>
  <tabstop>lineEdit_data</tabstop>
  <tabstop>pushButton_novo_2</tabstop>
  <tabstop>pushButton_gravar_2</tabstop>
  <tabstop>pushButton_voltar_2</tabstop>
  <tabstop>lineEdit_estoque_quantidade</tabstop>
  <tabstop>tableWidget_produtos</tabstop>
  <tabstop>pushButton_gravar_produto</tabstop>
  <tabstop>pushButton_remover</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
